CONTRIBUTING.md

```diff
@@ -1,12 +1,46 @@
 # CONTRIBUTING
 
-Welcome to dash-canvas!
+Welcome to dash-canvas! Contributions, bug reports, comments about the
+package are very welcome. The project is hosted on
+https://github.com/plotly/dash-canvas. 
 
-## Structure of the code
+This project was initially generated by the
+[dash-component-boilerplate](https://github.com/plotly/dash-component-boilerplate),
+which contains the minimal set of code required to create your own custom
+Dash component.
 
-React component DashCanvas: `src/lib/components/DashCanvas.react.js`
+## Structure of the code:
 
-Build Javascript component: `npm run build:all` at the root (see https://dash.plot.ly/react-for-python-developers for more details).
+- the source code of the ``dash_canvas.DashCanvas`` canvas reactive
+  object is the JavaScript React class in
+``src/lib/components/DashCanvas.react.js``. If you are not familiar with
+React and JavaScript, [this
+tutorial](https://dash.plot.ly/react-for-python-developers) is a good
+primer. In order to build the Javascript component: excecute at the root
+of the package
```

Comment:
Do you also want to add a comment about installing the Python package? (e.g. `pip install -e .` or equivalent)

Reply:
Good idea. Since I'm terrified by Javascript but fluent with Python I assumed it would be the same for other contributors but it could be the other way around :-)